examples/inverse/plot_lcmv_beamformer.py

```diff
@@ -56,11 +56,13 @@
 forward = mne.read_forward_solution(fname_fwd)
 forward = mne.convert_forward_solution(forward, surf_ori=True)
 
-# Compute regularized noise and data covariances
-noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0, method='shrunk',
-                                   rank=None)
+# Compute noise and data covariances. For beamformers, you want to use the
+# empirical covariance matrix. Regularization will be applied by the
+# beamformer.
```

Comment:
If possible can you clarify why it's better to do it manually in the beamformer rather than let the advanced, cross-validated estimators (e.g., `shrunk`) automatically choose the best reg, or even manually do `mne.cov.regularize`?

If it's not clear why it works but you have experienced that it does, I'm okay with merging this if you can add a bullet in #5365 to investigate it.

Reply:
How's this?